src/commons/api/ca.rs

```diff
@@ -141,23 +141,55 @@ impl From<&IdCert> for IdCertPem {
     }
 }
 
+//------------ ChildState ----------------------------------------------------
+
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
This clippy directiver is redundant here, right?

Reply:
indeed, removed